CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
   * `http.StatusServiceUnavailable` (503) and `codes.Unknown` are replaced with `codes.Internal`.
 * [CHANGE] Upgrade Node.js to v20. #6540
 * [CHANGE] Querier: `cortex_querier_blocks_consistency_checks_failed_total` is now incremented when a block couldn't be queried from any attempted store-gateway as opposed to incremented after each attempt. Also `cortex_querier_blocks_consistency_checks_total` is incremented once per query as opposed to once per attempt (with 3 attempts). #6590
+  [CHANGE] Query-Scheduler: introduce tree queue structure to support round-robin fairness across multiple queuing dimensions. #6533
```

Comment:
I don't think this warrants a changelog entry - we usually only add changelog entries for changes that are user-visible (eg. behaviour changes or performance improvements)

Reply:
more than happy to nix this anyway :+1: